views/metrilo-front-end-library.php

```diff
@@ -0,0 +1 @@
+<script type="text/javascript" src="<?php echo '//'. $this->tracking_library; ?>"></script>
```

Comment:
Follow up from: https://github.com/Metrilo/metrilo-woocommerce-integration/pull/49#discussion_r529532380

> This will add our library as a blocking javascript. We don't want our clients to suffer performance penalties if our javascript library is not served. We have a specific asynchronous way of adding the library in the old Woo plugin.

Reply:
again ^